spec/ruby_llm/tool_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'ruby_llm/tool'
```

Comment:
This shouldn't be required because `spec_helper.rb` imports RubyLLM already and Zeitwerk does the rest.